### DISCUSSION TITLE
[Neovim] I can't get to ":Copilot setup"

### DISCUSSION THREAD

**Discussion by aganomaminmi** [community/community#52909] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

Hi, I installed Copilot using `vim-plug`, but as soon as I install it I get the following error, and the same error when I run the Copilot setup command. I also tried `:Copilot! setup` but it was the same.

```
Client 1 quit with exit code 1 and signal 0
```
I am using nvim-lspconfig and nvim-lsp-installer, so I am wondering if the error is about that.

Then, when the `:Copilot` command is executed, the following log is output.
```
agent exited with status -1
Exception: copilot#agent: LSP client not available @ function copilot#Command[15]..copilot#Call[2]..<SNR>216_AgentCall[1]..<SNR>216_LspRequest, line 8
```

My setup:

macOS 13.2.1 on Intel Core i7
Node 16.15.0
Neovim 0.9.0
vim-plug 0.11.0

Do you have any suggestion, please? Thank you.

**Comment by justyn** [comment]
I'm seeing the same thing with [copilot.lua](https://github.com/zbirenbaum/copilot.lua) and Neovim 0.9.0 on Linux.

**Reply by justyn** [reply]
I had multiple node versions installed, and it turned out my nvim shell was loading the wrong version. I resolved this and copilot now runs.

**Reply by aganomaminmi** [reply]
Hi, @justyn.

Thanks for the info!

> I had multiple node versions installed, and it turned out my nvim shell was loading the wrong version. I resolved this and copilot now runs.

I tried one local node and using `copilot.lua` instead of `copilot.vim`, but the situation did not change.

Do you know how to output a detailed error log?
If you know, it would be great to know.

Thanks.

**Reply by justyn** [reply]
Hi @aganomaminmi I recommend you check the Neovim LSP error logs. I'm not entirely sure where they are on macOS but on my system it was `~/.local/state/nvim/lsp.log`

Also for reference I am now using node v18.16.0

Check you're running the right version of node in the neovim shell using `node --version`

**Reply by saeedtabrizi** [reply]
I have same issue with the following spec:
```
NVIM v0.10.0-dev
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3 
```
Is there any workaround or any patch for this issue ?
